osbs/api.py

```diff
@@ -589,6 +589,34 @@ def get_image_stream_tag(self, tag_id):
         return self.os.get_image_stream_tag(tag_id)
 
     @osbsapi
+    def put_image_stream_tag(self, tag_id, tag):
+        """Create or modify ImageStreamTag object
+
+        :param tag_id: str, name of ImageStreamTag prefixed by name of
```

Comment:
Nit: should "prefixed by" be "including"? The name of the ImageStreamTag is 'imagestream:tag'.